include/mapbox/geometry/feature.hpp

```diff
@@ -55,6 +55,10 @@ struct feature
     geometry_type geometry;
     property_map properties {};
     std::experimental::optional<identifier> id {};
+
+    // GCC 4.9 does not support C++14 aggregates with non-static data member
+    // initializers.
+    feature(geometry_type const& geometry_) : geometry(geometry_) {}
```

Comment:
How about:

```
feature(geometry_type geometry_,
        property_map properties_,
        std::experimental::optional<identifier> id_)
    : geometry(std::move(geometry_)),
      properties(std::move(properties_)),
      id(std::move(id_)) {}
```

Then all compilers can initialize with 1, 2, or 3 arguments, and `modernize-pass-by-value` is happy.